include/vrv/slur.h

```diff
@@ -191,13 +253,17 @@ class Slur : public ControlElement,
     //
 private:
     /**
-     * The drawing curve direction.
-     * This is calculated only when start - end points are on the same system. Otherwise
-     * it is left unset. This also means that it is reset only in ResetDrawing and not when
-     * the alignment is reset. The reason is because we want to preserve the value when the
-     * document is cast-off.
+     * The drawing curve direction
+     * This is calculated in the PrepareSlurs functor and contains an additional distinction
+     * for s-shaped slurs / mixed direction: whether the slur goes upwards or downwards
```

Comment:
* Why removing the entire previous code comment?
* Maybe we can drop the `SlurCurveDirection` enum if [this](https://github.com/music-encoding/music-encoding/issues/912) is agreed?

Reply:
- The previous code comment was not accurate anymore: the slur (even if it is broken) always relies on `m_drawingCurveDir` once `PrepareSlurs` was evaluated. This stores the drawing direction of the whole slur. The direction of the broken pieces is calculated via `Slur::CalcDrawingCurveDir`.
- Exactly, `SlurCurveDirection` should be temporary in the code as long as we cannot distinguish the S-slur direction via `curvature_CURVEDIR`. Once we have the extension on LibMEI, we can kick it out again.